responsibleai/tests/test_rai_insights_save_and_load_scenarios.py

```diff
@@ -265,14 +271,18 @@ def validate_rai_insights(
     test_data,
     target_column,
     task_type,
-    categorical_features
+    categorical_features,
+    feature_range_keys,
+    columns
 ):
-
     pd.testing.assert_frame_equal(rai_insights.train, train_data)
     pd.testing.assert_frame_equal(rai_insights.test, test_data)
     assert rai_insights.target_column == target_column
     assert rai_insights.task_type == task_type
     assert rai_insights.categorical_features == (categorical_features or [])
+    assert feature_range_keys.sort() == \
+        list(rai_insights._feature_ranges[0].keys()).sort()
```

Comment:
you probably want to validate the other metadata as well, like range type, min/max etc (both that keys exist and some value) to make sure the format is as expected

Reply:
I didn't do exact value comparison as it was producing different value for erroranalysis/countefactual at times.